### PR TITLE
Handle empty rect ROI

### PR DESCRIPTION
**Pull request by AnandBaburajan**
Fix for empty rectangular ROI in the GUI throwing `UnboundLocalError: local variable 'ymin' referenced before assignment`

## Contributor Checklist:

* [x] I have added/updated test cases

### CODE REVIEW

File: tests/test_masks.py [hunk 1]
```diff
@@ -100,3 +100,9 @@ def test_rectmask():
     assert np.allclose(m.rectangular(2, 2, -3, 3, 5, 5), rect3)
     assert np.allclose(m.rectangular(2, 2, 3, -3, 5, 5), rect2)
     assert np.allclose(m.rectangular(2, 2, -3, -3, 5, 5), rect1)
+
+
+def test_empty_rectmask():
+    assert not np.all(m.rectangular(2, 2, 0, 3, 5, 5))
+    assert not np.all(m.rectangular(2, 2, 3, 0, 5, 5))
+    assert not np.all(m.rectangular(2, 2, 0, 0, 5, 5))
```
Comment (sk1p) [inline tests/test_masks.py:108]: `not np.all` means that there is at least one false-y element in the array, is it possible that you mean `not np.any(...)` instead, as all elements are supposed to be `False`?

**Review by sk1p — COMMENTED** [review]
Good catch! I've added a comment about the test case, which I think can be improved. Thanks!

File: tests/test_masks.py (hunk 1 above)
Comment (AnandBaburajan) [reply tests/test_masks.py:108]: Yup, that's what I meant. Thanks!

**Review by sk1p — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**